server/queues/tasks/DetachDraftsFromCollectionTask.ts

```diff
@@ -0,0 +1,32 @@
+import { Op } from "sequelize";
+import { Collection, Document } from "@server/models";
+import BaseTask from "./BaseTask";
+
+type Task = {
+  collectionId: string;
+};
+
+export default class DetachDraftsFromCollectionTask extends BaseTask<Task> {
+  async perform(task: Task) {
+    const collection = await Collection.findByPk(task.collectionId, {
+      paranoid: false,
+    });
+
+    if (!collection || !collection.deletedAt) {
+      return;
+    }
+
+    await Document.update(
```

Comment:
I think we should query draft documents in the collection and loop over them calling `documentUpdater` here or the correct events / websocket messages won't be triggered.

Reply:
Sadly, neither `documentUpdater` nor `documentMover` possess the capability to _update `collectionId` to `null`_ so far :man_facepalming:

Reply:
Appropriate to bake this into `documentMover` since we're _moving out of collection_?

Reply:
Oh yea, that sounds good? We should generally have the ability to move back to drafts